src/python_helper/compare_helper.py

```diff
@@ -87,20 +87,25 @@ def check_function(self, function_name: str, old_package_name: str):
         best_match_package: str = self.find_best_matching_package(
             function_name)
         if best_match_package != old_package_name:
-            if self.mitigate_FPs(function_name, old_package_name, 1) == False:
+            if self.mitigate_FPs(function_name, old_package_name, accuracy) == False:
                 print(red("\nerror: "),
                       f"Function '{function_name}' in '{old_package_name}' package, may not be in the best matching package, consider placing it in '{best_match_package}' or choosing a better name for '{old_package_name}'!")
+                if nameSuggestions:
+                    print(green("\nsuggestion:"),
+                          "Top 5 highest matching package-names to function is: ", end="")
+                    self.list_most_similar(function_name)
 
-    def list_most_similar(self):
+    def list_most_similar(self, function):
         """
         Takes function_list and for each function
         checks for most similar words in training data,
 
         prints list of most similar word.
         """
-        for function in self.function_list:
-            print(blue(f'Results for words found similar to "{function}"'))
-            print(green(self.model.wv.most_similar(self.sanitize_name(function))))
+        most_similar = self.model.wv.similar_by_key(self.sanitize_name(function), topn=5)
```

Comment:
Very nice addition 👍 but do we know that we won't get function names back as matches here? And not always package names?

Reply:
Also what is the difference between `most_similair` and `similair_by_key` ?

Reply:
to add to 1. theoretically it should be packages as those are the closest. So itäs useful to a certain degree.

Reply:
to add to 2. it returns a list of tuples by (string, distance), while most_similar returns (unknown, distance). It supports better typing.

Reply:
> to add to 1. theoretically it should be packages as those are the closest. So itäs useful to a certain degree.

Guess a list of all trained on packages would be nice to use as a filter. But for the future...

Guess we just have to be honest about this in the output, so users understand that limitation and don't get confused, assume it's a big bug.